### PR TITLE
Support dynamic non-root user for fuse

### PR DESCRIPTION
**Pull request by TrafalgarZZZ**
### Ⅰ. Describe what this PR does
Support dynamic non-root user for fuse

Alluxio-fuse now can be run as a non-root user by setting `spec.runAs` in an `AlluxioRuntime` CRD yaml.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: charts/alluxio/templates/fuse/daemonset.yaml [hunk 1]
```diff
@@ -67,6 +65,29 @@ spec:
...
+            runAsUser: 0
+            runAsGroup: 0
+            privileged: true
+            capabilities:
+              add:
+                - SYS_ADMIN
+          volumeMounts:
+            - name: alluxio-fuse-mount
+              mountPath: {{ .Values.fuse.mountPath | dir }}
+              mountPropagation: Bidirectional
```
Comment (cheyang) [inline charts/alluxio/templates/fuse/daemonset.yaml:89]: mountPropagation: Bidirectional 为什么必须要加？

File: charts/alluxio/templates/fuse/daemonset.yaml [hunk 2]
```diff
@@ -101,6 +122,8 @@ spec:
             value: "{{ $value }}"
           {{- end }}
           securityContext:
+            runAsUser: {{ .Values.fuse.user }}
+            runAsGroup: {{ .Values.fuse.group }}
```
Comment (cheyang) [inline charts/alluxio/templates/fuse/daemonset.yaml:126]: 是否可以简化成.Values.user ？

File: samples/ufspath/dataset.yaml [hunk 3]
```diff
@@ -49,3 +49,6 @@ spec:
     alluxio.user.local.reader.chunk.size.bytes: 256MB
     alluxio.worker.network.reader.buffer.size: 256MB
     alluxio.user.streaming.data.timeout: 300sec
+  fuse:
+    image: registry.cn-huhehaote.aliyuncs.com/alluxio/alluxio-fuse
+    imageTag: 2.3.0-SNAPSHOT-f0b220e-nonroot
```
Comment (cheyang) [inline samples/ufspath/dataset.yaml:54]: 我构建一个最新的镜像，我们把内置的镜像也替换掉吧。

**Review by cheyang — CHANGES_REQUESTED** [review]

File: samples/ufspath/dataset.yaml (hunk 3 above)
Comment (TrafalgarZZZ) [reply samples/ufspath/dataset.yaml]: done

File: charts/alluxio/templates/fuse/daemonset.yaml (hunk 2 above)
Comment (TrafalgarZZZ) [reply charts/alluxio/templates/fuse/daemonset.yaml]: done

File: charts/alluxio/templates/fuse/daemonset.yaml (hunk 1 above)
Comment (TrafalgarZZZ) [reply charts/alluxio/templates/fuse/daemonset.yaml]: 不需要加，已经去除

**Pull request merged**